### ISSUE TITLE
Failed to create symbolic link

### ISSUE THREAD

**Issue by amsolucionesweb** [deployphp/deployer#434]
After update form 3.0.6 to 3.0.7 version when deploy I get 

``` [RuntimeException]                                                                                                                                         
  ln: failed to create symbolic link `/var/www/vhosts/site.com/httpdocs/releases/20150819144214/app/config/parameters.yml': No such file or directory 
```
```
Exception trace:
 () at phar:///usr/local/bin/dep/src/Server/Remote/PhpSecLib.php:114
 Deployer\Server\Remote\PhpSecLib->run() at phar:///usr/local/bin/dep/src/functions.php:296
 run() at phar:///usr/local/bin/dep/recipe/common.php:192
 {closure}() at n/a:n/a
 call_user_func() at phar:///usr/local/bin/dep/src/Task/Task.php:73
 Deployer\Task\Task->run() at phar:///usr/local/bin/dep/src/Executor/SeriesExecutor.php:39
 Deployer\Executor\SeriesExecutor->run() at phar:///usr/local/bin/dep/src/Console/TaskCommand.php:82
 Deployer\Console\TaskCommand->execute() at phar:///usr/local/bin/dep/vendor/symfony/console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:878
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:195
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:126
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/dep/src/Deployer.php:99
 Deployer\Deployer->run() at phar:///usr/local/bin/dep/bin/dep:62
 require() at /usr/local/bin/dep:4
```

When I rollback to version 3.0.6 the deploy goes fine

**Comment by elfet** [comment]
Wow! Need to investigate what happens. @oanhnn does your deploys work fine with 3.0.6 and 3.0.7?

**Comment by oanhnn** [comment]
@amsolucionesweb We need more information to reproduction this error.
I think connection was lost by timeout same problem of @AlexStansfield in #429

**Comment by AlexStansfield** [comment]
yeah, this is exactly what I was getting.

**Comment by amsolucionesweb** [comment]
Now, after run ```dep self-update``` and versión 3.0.8 is installed when run ```dep deploy prod``` I get

```
PHP Notice:  Cannot connect to linux.server.com.ar:22. Error 110. Connection timed out in phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php on line 1011
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/dep:0
PHP   2. require() /usr/local/bin/dep:4
PHP   3. Deployer\Deployer->run() phar:///usr/local/bin/dep/bin/dep:62
PHP   4. Symfony\Component\Console\Application->run() phar:///usr/local/bin/dep/src/Deployer.php:99
PHP   5. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:126
PHP   6. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:195
PHP   7. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:878
PHP   8. Deployer\Console\TaskCommand->execute() phar:///usr/local/bin/dep/vendor/symfony/console/Command/Command.php:259
PHP   9. Deployer\Executor\SeriesExecutor->run() phar:///usr/local/bin/dep/src/Console/TaskCommand.php:82
PHP  10. Deployer\Task\Task->run() phar:///usr/local/bin/dep/src/Executor/SeriesExecutor.php:39
PHP  11. call_user_func() phar:///usr/local/bin/dep/src/Task/Task.php:73
PHP  12. {closure:phar:///usr/local/bin/dep/recipe/common.php:67-93}() phar:///usr/local/bin/dep/src/Task/Task.php:73
PHP  13. Deployer\Server\Remote\PhpSecLib->connect() phar:///usr/local/bin/dep/recipe/common.php:68
PHP  14. phpseclib\Net\SFTP->login() phar:///usr/local/bin/dep/src/Server/Remote/PhpSecLib.php:54
PHP  15. call_user_func_array() phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP.php:392
PHP  16. phpseclib\Net\SSH2->_login() phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP.php:392
PHP  17. phpseclib\Net\SSH2->_connect() phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php:1879
PHP  18. user_error() phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php:1011

                                               
  [RuntimeException]                              
  Unable to login with the provided credentials.  

```

My server config is:

```php
server('prod', 'linux.server.com.ar', 22)
    ->env('deploy_path', '/var/www/vhosts/server.com.ar/httpdocs')
    ->user('user')
    ->password()
    ->stage('prod');
```

The credentials was not changed and I can login via ssh normally.

**Comment by AlexStansfield** [comment]
This is an issue introduced in 3.0.8, it should be fixed by this #435

**Comment by elfet** [comment]
Fixed.
